Update CODE_OF_CONDUCT.md (#22781)
grafana/grafana: CODE_OF_CONDUCT.md

## Our Pledge

> In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion,  sexual identity and orientation, or other factors.

Reviewer: This is pretty vague. Who decides what those other factors are?

Author: Conduct@ like with everything else; it seems counter-intuitive to have a limited list.

I have no issue reverting this.

Reviewer 2: Agreed. We do not wish to be vague on this.

## Our Standards

> Examples of behavior which contributes to creating a positive environment include:

Reviewer: I think "that" is correct, as I believe this is a [defining clause](https://www.grammarly.com/blog/which-vs-that/)

Author: @oddlittlebird ?

Reviewer 2: Both are nominally correct, I prefer "that," because it is less ambiguous. When in doubt, refer to the Microsoft Style Guide: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which

## Enforcement

> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the Grafana Labs conduct team at conduct@grafana.com. The conduct team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.

Reviewer 2: ```suggestion
Report instances of abusive, harassing, or otherwise unacceptable behavior by contacting the Grafana Labs conduct team at conduct@grafana.com. The conduct team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team must maintain confidentiality regarding the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
